client/src/components/molecules/NotificationPopup.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <q-btn
+    flat
+    padding="none"
+    data-cy="dropdown_notificiations"
+    class="notifications-dropdown"
+    :aria-label="$t('header.notification_button')"
+  >
+    <q-icon name="notifications" />
+    <q-badge floating color="light-blue-3" rounded />
+    <q-popup-proxy
+      ref="popupProxy"
+      v-model="isVisible"
+      max-width="400px"
+      position="top"
+    >
+      <div class="notifications-container">
+        <q-list
+          role="navigation"
+          aria-label="Dropdown Navigation"
+          bordered
+          separator
+          class="notifications-list"
+        >
+          <notification-list-item
+            v-for="(item, index) in items"
+            :key="index"
+            :note="item"
+            clickable
+            :class="{ unread: !item.viewed }"
+          />
+        </q-list>
+        <q-btn-group spread>
+          <q-btn to="/feed">View More</q-btn>
+          <q-btn>Dismiss All</q-btn>
+        </q-btn-group>
+      </div>
+    </q-popup-proxy>
+  </q-btn>
+</template>
+
+<script>
+import { defineComponent, ref, watch, nextTick } from "@vue/composition-api"
+import { notificationItems } from "src/graphql/fillerData"
+import NotificationListItem from "src/components/atoms/NotificationListItem.vue"
+
+/**
+ * Notification Dropdown menu
+ */
+export default defineComponent({
+  name: "NotificationPopup",
+  components: { NotificationListItem },
+  setup() {
+    const popupProxy = ref(null)
+    const isVisible = ref(false)
+
+    watch(isVisible, (newValue) => {
+      if (newValue === false) {
+        return
+      }
+      nextTick(() => {
+        popupProxy.value.$refs.popup.$children[0].$el.id =
+          "notifications-wrapper"
+      })
+    })
+    return { items: notificationItems, isVisible, popupProxy }
+  },
+})
+</script>
+
+<style lang="sass">
+.q-dialog__inner .notifications-container .q-list.notifications-list
+  width: auto
+  .q-item
+    width: 100%
+
+#notifications-wrapper.q-menu
+  display: flex
+  overflow: hidden
+  flex-flow: column
+.notifications-container
+  background: white
+  display: flex
+  flex-flow: column
+  overflow: hidden
+  .q-list.notifications-list
+    display: flex
+    flex-flow: column wrap
+    overflow: hidden
+    .q-item
+      min-height: 40px
+      padding: 2px 16px 2px 0px
+      width: 100%
+      margin-right: 10px
```

Comment:
I'll take a deeper look at this tomorrow to see if all the styles here are necessary.